docs/docs/explanations/configuration-best-practices.mdx

```diff
@@ -0,0 +1,65 @@
+---
+title: Configuration best practices
+id: config-best-practices
+description: A guide to best practices for configuring Nebari
+---
+
+This document highlights and consolidates configuration best practices that are introduced throughout the Getting Started, How To's, and Advanced Settings sections of the documentation.
+
+### Naming conventions
+
+When Nebari creates a new cloud resource on each one of the cloud providers, it will have to provide properties such as a name, project id, buckets, etc... for these resources.
```

Comment:
```suggestion
When Nebari creates a new cloud resource on each one of the cloud providers, it will have to provide properties such as a name, project id, buckets, etc. for these resources.
```